js/toolbar.js

```diff
@@ -232,7 +238,10 @@ class Toolbar {
                 ["ar", _("عربى"), "innerHTML"],
                 ["te", _("తెలుగు"), "innerHTML"],
                 ["he", _("עִברִית"), "innerHTML"],
-                ["ur", _("اردو"), "innerHTML"]
+                ["ur", _("اردو"), "innerHTML"],
+                ["light", _("Light Mode"), "innerHTML"],
+                ["dark", _("Dark Mode"), "innerHTML"]
+                // ["custom", _("Theme Name"), "innerHTML"],
```

Comment:
We should be consistent with the string (even if it is commented out). How about "Custom Theme" everywhere?